joplin/js/editor.js

```diff
@@ -203,46 +169,43 @@ $(function() {
     replaceLanguageLabels();
 
     // TODO: refactor into a function, evaluate performance
-    // have better variable namepsace seperation
     let labelList = state.languageLabels;
-    for (let label in labelList) {
-      if (labelList[label].querySelector) {
-        let languageTag = labelList[label].querySelector('span').innerText;
-        // these seem to be nested twice, from the title to the containing element
-        // TODO: come up with a more elegant and maintable way to check what elements ought to be hidden
+    for (let label of labelList) {
```

Comment:
Just curious/trying to learn, why did we switch this from a 'for in' loop to a 'for of' loop?

Reply:
great question! I learned something new working on this about for in vs for of. Basically what happened when I did this: ` for (let label in labelList) { console.log(label) }` what was printed was the numbers 1-40 and *then* things like "length, prevObject, jquery, constructor, toArray, get, pushStack, each, map" and a lot more. 

So then I was doing some reading as to what was happening and found this: https://thecodebarbarian.com/for-vs-for-each-vs-for-in-vs-for-of-in-javascript.html

if we use `for label of labellist` we can just access the label directly instead of using an iterator and `labelList[label]`